mergify_engine/actions/copy.py

```diff
@@ -35,12 +35,52 @@ def Regex(value):
 class CopyAction(actions.Action):
     KIND = "copy"
     SUCCESS_MESSAGE = "Pull request copies have been created"
+    FAILURE_MESSAGE = "No copy have been created"
 
     validator = {
         voluptuous.Required("branches", default=[]): [str],
         voluptuous.Required("regexes", default=[]): [Regex],
     }
 
+    def _copy(self, pull, branch_name):
+        """Copy the PR to a branch.
+
+        Returns a tuple of strings (state, reason).
+        """
+        try:
+            branch = pull.g_pull.base.repo.get_branch(parse.quote(branch_name, safe=""))
+        except github.GithubException as e:
```

Comment:
Not directly related to the change, but status >= 500 should be reraised here.

Reply:
I imagine returning `None` as state is better since we can then retry, right?
Doing that.